test/RpcCompatibility.js

```diff
@@ -243,6 +243,19 @@ describe('RpcCompatibility', () => {
             checkBlock(block, false);
             checkBlockTimeStamp(parseInt(block.timestamp, 16), 12000);
         });
+
+        it('blocks have valid parent hashes', async () => {
```

Comment:
Add another test for finalized block (if the chain supports it, that one should work too)

Reply:
sg 75eeb36acf34872bc010efaf682d245903bb553d